doc/dev/getting-started.rst

```diff
@@ -428,67 +333,3 @@ Here are a few suggestions:
 
 * You can also help other people out by watching for new issues or
   looking at pull requests.  Remember to be nice and polite!
-
-Your second contribution...
```

Comment:
I think the git commands here should be kept in the beginner instructions - let people know how to reset their repo after their contribution is merged.  Otherwise you end up with messiness.